pyro/contrib/timeseries/gp.py

```diff
@@ -97,6 +97,7 @@ def _forecast(self, dts, filtering_state, include_observation_noise=True):
         assert dts.dim() == 1
         dts = dts.unsqueeze(-1).unsqueeze(-1).unsqueeze(-1)
         trans_matrix, process_covar = self.kernel.transition_matrix_and_covariance(dt=dts)
+        trans_matrix = trans_matrix[..., 0:1]
```

Comment:
Are you intentionally sneaking semantics changes into a docs PR?

Reply:
yes except i flagged this optimization with a comment